### ISSUE TITLE
Tusimple数据训练结果不太好是超参还是data问题？

### ISSUE THREAD

**Issue by pauperonway** [MaybeShewill-CV/lanenet-lane-detection#338]
按照readme进行训练，没有修改参数，修改epoch为120K，val选择train.txt的最后600个case，所以train的数量大概是3000个。训练的二值图loss如下图：
![企业微信截图_15784684458763](https://user-images.githubusercontent.com/3377021/71959105-e202b700-322c-11ea-9aac-33ac2008f4a9.jpg)

**Comment by pauperonway** [comment]
@MaybeShewill-CV 测试您的训练参数效果要比我自己训练的好很多，由于FP一直降不下去，所以很多误检。
<img width="208" alt="企业微信截图_15784684458763" src="https://user-images.githubusercontent.com/3377021/71959344-7d942780-322d-11ea-83b4-7bafdd704005.png">

**Comment by pauperonway** [comment]
和这个问题一样 https://github.com/MaybeShewill-CV/lanenet-lane-detection/issues/247

**Comment by pauperonway** [comment]
我把作者之前的一次提交pull下来啊，正在训练，明天看一下结果。
版本信息：
commit a9650d2fe6f2ceab9e83d8e1eb0ab875a0787f58
Author: luoyao <luoyao@baidu.com>
Date:   Tue May 29 14:52:34 2018 +0800

    add source image data

**Comment by pauperonway** [comment]
训练到8k左右出现NAN，可能是大家之前说的L1 L2的问题，我改一下代码继续
info：
 Epoch: 8356 total_loss= 0.069291 binary_seg_loss= 0.043981 instance_seg_loss= 0.094601 accuracy= 0.981102 mean_cost_time= 0.127303s
E0109 06:29:44.031018 43628 train_lanenet.py:226] cost is: nan
E0109 06:29:44.031258 43628 train_lanenet.py:227] binary cost is: nan
E0109 06:29:44.031470 43628 train_lanenet.py:228] instance cost is: nan

**Comment by pauperonway** [comment]
<img width="193" alt="111" src="https://user-images.githubusercontent.com/3377021/72044332-7f71ef80-32ee-11ea-8df3-cd31b9f7c9ae.png">

**Comment by pauperonway** [comment]
作者改过loss及accuracy的计算方式，所以loss图的数值会有不一样，不过我训练的模型确实没有作者的好，哭。再读读代码

**Comment by pauperonway** [comment]
batchsize设置为8，训练出的结果要比4好一下，FP会降低。

**Comment by MaybeShewill-CV** [comment]
@pauperonway batch size还是尽量大比较好 我是gtx 1070ti 显存比较小:）

**Comment by xxKomorebixx** [comment]
> @MaybeShewill-CV 测试您的训练参数效果要比我自己训练的好很多，由于FP一直降不下去，所以很多误检。
> <img alt="企业微信截图_157846930922" width="207" src="https://user-images.githubusercontent.com/3377021/71959589-0b701280-322e-11ea-8d72-5215677046ea.png">

你好，请问下fp是在哪里看的呢？我的tensorboard里只有loss和miou，是添加了代码吗？
